pkg/kubectl/cmd/util/editor/editor.go

```diff
@@ -136,10 +137,10 @@ func (e Editor) Launch(path string) error {
 	return nil
 }
 
-// LaunchTempFile reads the provided stream into a temporary file in the given directory
-// and file prefix, and then invokes Launch with the path of that file. It will return
-// the contents of the file after launch, any errors that occur, and the path of the
-// temporary file so the caller can clean it up as needed.
+// LaunchTempFile reads the provided stream into a temporary file with prefix and suffix
+// in the given directory and file prefix, and then invokes Launch with the path of that file.
+// It will return the contents of the file after launch, any errors that occur, and the path
+// of the temporary file so the caller can clean it up as needed.
```

Comment:
same with named result parameters. `[]byte`, and `string` are not specific enough to be self-explanatory.